boards/arm/96b_carbon/doc/96b_carbon.rst

```diff
@@ -281,6 +317,51 @@ terminal:
 
    Hello World! arm
 
+.. _96b_carbon_verify_bluetooth:
+
+Verifying Bluetooth Functionality
+---------------------------------
+
+This section contains instructions for verifying basic Bluetooth
+functionality on the board. For help on Zephyr applications
+in general, see :ref:`build_an_application`.
+
+1. Make sure you have flashed the nRF51 with the hci_spi sample
+   application as described in :ref:`96b_carbon_nrf51_bluetooth`.
+
+2. Make sure you have dfu-util installed as described earlier on this
+   page.
+
+3. Build the ``samples/bluetooth/ipsp`` application for 96b_carbon::
+
+     $ cd <zephyr_root_path>
+     $ source zephyr-env.sh
+     $ cd $ZEPHYR_BASE/samples/bluetooth/ipsp/
+     $ make BOARD=96b_carbon
+
+4. Flash the compiled application using dfu-util. See the instructions
+   above for how to put your board into DFU mode if you haven't done
+   this before::
+
+     $ make BOARD=96b_carbon flash
+
+Following that, refer to the instructions in
```

Comment:
Since this is another step, use one more numbered list item:

     5. Refer to the instructions in ...